Source/WebCore/accessibility/AXTextMarker.cpp

```diff
@@ -1579,9 +1579,102 @@ bool AXTextMarker::equivalentTextPosition(const AXTextMarker& other) const
     return objectID() != other.objectID() && (findMarker(AXDirection::Next, CoalesceObjectBreaks::No, IgnoreBRs::Yes) == other || findMarker(AXDirection::Previous, CoalesceObjectBreaks::No, IgnoreBRs::Yes) == other);
 }
 
-std::partial_ordering AXTextMarker::partialOrderByTraversal(const AXTextMarker& other) const
+// This function implements a fast way to determine our ordering relative to |other| is to find
```

Comment:
Nit: sentence is unclear, how about:

// This function implements a fast way to determine our ordering relative to |other|: find...

Reply:
Whoops, yes this got garbled after I rewrote it once! Taking your suggestion in the latest commit.